teachable.rb

```diff
@@ -0,0 +1,30 @@
+module Teachable
+  def offer_high_five
```

Comment:
Why is High Five here?  It a student behavior as well.  What can you do with it instead?